service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NewHearingsAddedEventHandlerTest.java

```diff
@@ -0,0 +1,131 @@
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.NOTICE_OF_NEW_HEARING;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.CAFCASS_EMAIL_ADDRESS;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_CODE;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_EMAIL_ADDRESS;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBooking;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    NewHearingsAddedEventHandler.class,
+    JacksonAutoConfiguration.class,
+    LookupTestConfig.class,
+    FixedTimeConfiguration.class})
+class NewHearingsAddedEventHandlerTest {
+    private static final String CASE_REFERENCE = "12345";
+
+    @Autowired
+    private NewHearingsAddedEventHandler newHearingsAddedEventHandler;
+
+    @Autowired
+    private Time time;
+
+    @MockBean
+    private NotificationService notificationService;
+
+    @MockBean
+    private NewNoticeOfHearingEmailContentProvider newNoticeOfHearingEmailContentProvider;
+
+    @MockBean
+    private HearingBookingService hearingBookingService;
+
+    @MockBean
+    private InboxLookupService inboxLookupService;
+
+    @MockBean
+    private RepresentativeNotificationService representativeNotificationService;
+
+    private LocalDateTime futureDate;
+
+    @BeforeEach
+    void setUp() {
+        futureDate = time.now().plusDays(1);
+    }
+
+    @Test
+    void shouldSendNotificationToLACafcassAndRepresentativesWhenNewHearingIsAdded() {
+        final ObjectMapper mapper = new ObjectMapper();
+        final CallbackRequest callbackRequest = callbackRequest();
+        final EventData eventData = new EventData(new NewHearingsAddedEvent(callbackRequest));
+        final CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+        List<Element<HearingBooking>> hearingBookings = List.of(
+            element(UUID.randomUUID(), createHearingBooking(futureDate.plusDays(5), futureDate.plusDays(6))));
+        NewNoticeOfHearingTemplate newNoticeOfHearingTemplate =
+            NewNoticeOfHearingTemplate.builder().build();
+
+        given(hearingBookingService.getSelectedHearings(any(), any())).willReturn(hearingBookings);
+        given(inboxLookupService.getNotificationRecipientEmail(caseDetails, LOCAL_AUTHORITY_CODE))
+            .willReturn(LOCAL_AUTHORITY_EMAIL_ADDRESS);
+        given(newNoticeOfHearingEmailContentProvider.buildNewNoticeOfHearingNotification(
+            any(CaseDetails.class), any(HearingBooking.class), any()))
```

Comment:
On unit test level we should try to avoid using "any()" we now what class will be passed here we should validate. 

I can pass the wrong class in the code and unit test won't pass.